### ISSUE TITLE
[BUG] init wpbrowser error: Unexpected characters near "    " at line 20 (near "title: "Test"        ")

### ISSUE THREAD

**Issue by dale42** [lucatume/wp-browser#474]
**tl;dr**

Running `vendor/bin/codecept init wpbrowser` and answering the prompts produces the error message:
```
Unexpected characters near "    " at line 20 (near "title: "Test"        ")
```

I believe the problem is white space on line 619 of src/Codeception/Template/Wpbrowser.php

**Environment**

|   |   |
|---|---|
| OS:                            | macOS 10.15.7 |
| PHP version:                   | 7.4.12 |
| Composer version               | 2.0.1 |
| Installed Codeception version: | 4.1.12 |
| Installed wp-browser version:  | 3.0.1 |
| WordPress version:             | n/a |
| Local dev:                     | PHP built-in server |
| WP structure and management:   | n/a |

composer.json:
```yml
{
    "name": "dale/bug-demo",
    "require": {},
    "require-dev": {
        "lucatume/wp-browser": "^3.0",
        "codeception/module-asserts": "^1.3",
        "codeception/module-phpbrowser": "^1.0",
        "codeception/module-webdriver": "^1.1",
        "codeception/module-db": "^1.0",
        "codeception/module-filesystem": "^1.0",
        "codeception/module-cli": "^1.1",
        "codeception/util-universalframework": "^1.0"
    }
}
```

**Did you use the `codecept init wpbrowser` command?**

Yes

**Did you take a look at Codeception and wp-browser documentation?**

Yes

**Codeception configuration file**

Unable to complete the setup to generate the file

**Suite configuration file**

Unable to complete setup to generate the file

**Describe the issue you're encountering**

Running `vendor/bin/codecept init wpbrowser` and answering the prompts produces the error message:
```
Unexpected characters near "    " at line 20 (near "title: "Test"        ")
```

**Output**
```
$ vendor/bin/codecept init wpbrowser
Welcome to wp-browser, a complete testing suite for WordPress based on Codeception and PHPUnit!
This command will guide you through the initial setup for your project.

If you are new to wp-browser please take the time to read this guide:
https://github.com/lucatume/wp-browser#initial-setup

? <warning>I acknowledge wp-browser should run on development servers only, that I have made a backup of my files and database contents before proceeding.</warning> (y/n) y


 Bootstrapping Codeception for WordPress

File codeception.yml created       <- global configuration

? Would you like to set up the suites interactively now? (yes)
 ---

---

 WPLoader and WordPress modules need to access the WordPress files to work.

? Where is WordPress installed? (/var/www/html)
? What is the path, relative to WordPress root URL, of the admin area of the test site? (/wp-admin)

 The WPDb module needs the database details to access the test database used by the test site.

? What is the name of the test database used by the test site? (test)
? What is the host of the test database used by the test site? (localhost)
? What is the user of the test database used by the test site? (root)
? What is the password of the test database used by the test site?
? What is the table prefix of the test database used by the test site? (wp_)

 WPLoader will reinstall a fresh WordPress installation before the tests.
It needs the details you would typically provide when installing WordPress from scratch.

 WPLoader should be configured to run on a dedicated database!
The data stored on the database used by the WPLoader module will be lost!

? What is the name of the test database WPLoader should use? (test)
? What is the host of the test database WPLoader should use? (localhost)
? What is the user of the test database WPLoader should use? (root)
? What is the password of the test database WPLoader should use?
? What is the table prefix of the test database WPLoader should use? (wp_)
? What is the URL the test site? (http://wordpress.test)
? What is the email of the test site WordPress administrator? (admin@wordpress.test)
? What is the title of the test site? (Test)
? What is the login of the administrator user of the test site? (admin)
? What is the password of the administrator user of the test site? (password)
? Are you testing a plugin, a theme or a combination of both (both)? (plugin)
? What is the folder/plugin.php name of the plugin? (my-plugin/my-plugin.php)
? Does your project needs additional plugins to be activated to work? (no)
 Unit helper has been created in tests/_support/Helper
 UnitTester actor has been created in tests/_support
 Actions have been loaded
tests/unit created                 <- unit tests
tests/unit.suite.yml written       <- unit tests suite configuration
 Wpunit helper has been created in tests/_support/Helper

In Parser.php line 768:

  Unexpected characters near "    " at line 20 (near "title: "Test"        ").


init [--path PATH] [--namespace [NAMESPACE]] [--] <template>

```
**To Reproduce**

```shell
composer init
composer require --dev lucatume/wp-browser
composer require --dev codeception/module-asserts codeception/module-phpbrowser codeception/module-webdriver codeception/module-db codeception/module-filesystem codeception/module-cli codeception/util-universalframework
vendor/bin/codecept init wpbrowser
```

**Screenshots**

<img width="604" alt="cli-error" src="https://user-images.githubusercontent.com/764120/101419095-2155df80-38a4-11eb-81a3-6009b82cfd87.png">

**Additional context**

I solved the problem in my installation by removing the space characters on line 619 of src/Codeception/Template/Wpbrowser.php

<img width="693" alt="whitespace" src="https://user-images.githubusercontent.com/764120/101418934-c623ed00-38a3-11eb-893b-12ec267e7f38.png">

**Comment by dale42** [comment]
Have created a pull request: fix: Remove space characters causing yml parse error #475

**Comment by lucatume** [comment]
@dale42 thanks for taking the time to report the issue, provide the information and provide a PR as well.

I've "wrapped" your PR into one of mine to check on the build and add a changelog entry.

**Comment by lucatume** [comment]
@dale42 I've merged your PR, thanks.

I'm closing this now, let me know if there are any further issues with this.
